jbpm-container-test/jbpm-in-container-test/shrinkwrap-war-profiles/pom.xml

```diff
@@ -215,7 +205,7 @@
         </dependency>
         <dependency>
           <groupId>org.jbpm</groupId>
-          <artifactId>idep-hibernate-entitymanager</artifactId>
+          <artifactId>idep-hibernate-core</artifactId>
```

Comment:
If we change this for simple hibernate-core artifact, then idep-hibernate-entitymanager will not be used anywhere. The main reason why we had a wrapper for it was that before we used it directly with shrinkwrap when building an archive. But since now this is a part of tomcat-specific profile, we don't need a nice wrapper around it. What do you think?

Reply:
For the tomcat profile, I agree, we may directly include the hibernate-core dependency, without wrapper. I chose this option for changing as less as possible